src/Symfony/Component/Messenger/Transport/Schedule/JobScheduleInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Schedule;
+
+interface JobScheduleInterface
```

Comment:
As jobs are messages (as we call them in Messenger), what about renaming this interface to `ScheduledMessageInterface`?

Reply:
Well, it is not a message itself. It's a scheduled task that provides messages (by `getJob`) on time when it's needed to be done. Maybe `ScheduledTaskInterface` and `getMessage()` method? Then `PeriodicalTask` which implements `ScheduledTaskInterface`

Reply:
I used "job" over `task` because the word "task" is heavily overused.

Reply:
Well, I have renamed it as you advised :)